src/zarpaint/_copy_data.py

```diff
@@ -14,4 +15,5 @@ def copy_data(
     ndim_dst = dst_data.ndim
     slice_ = napari_viewer.dims.current_step
     slicing = slice_[:ndim_dst - ndim_src]
-    dst_data[slicing] = src_data
\ No newline at end of file
+    dst_data[slicing] = np.uint32(src_data)
```

Comment:
Hey @Kaltzisp! I can't reproduce whatever problem this might have been solving. Copying across dtypes works fine, even if src_data is a dask array. Could you please remove this line until we can reproduce the problem? As well as the numpy import. The refresh is handy so I'll merge that!